### ISSUE TITLE
Certain keys not being passed through

### ISSUE THREAD

**Issue by a-singer** [EscolarProgramming/NVDARemotePublic#1]
Hello,

When I connect to the remote server, whether VoiceOver is enabled or disabled, certain keys I press on the keyboard are not passed through to the remote side. For example, I am unable to send through the play/pause key, the volume key, the home key, etc. These are all received by the iDevice, not by the remote computer. I understand that it may be impossible to have these keys sent to the remote machine but thought I would raise this, as on the PC, all keyboard keys I can think of are passed through to the remote side.

Thanks,

**Comment by EscolarProgramming** [comment]
Hello,
Thanks for the feedback. Unfortunately, it is like you already mentioned. This buttons are catched by iOS and my app doesn't notice it. This seems to be an Apple restriction.

**Comment by bdorer** [comment]
hi, in this case you should specify which keys are passed through. I tried just before and even arrow keys were used for VO and not passed to NVDA. This is also true for most other keys. I'm using NVDA 2020.4 with latest stable of nvdaremote. My iPhone SE runs iOS 14.5. Should I open a new issue?

**Comment by a-singer** [comment]
Hi,

In my experience, VO commands are not passed through, so turning off quicknav, for example, will get the arrows to be passed through. The same may be said of turning VO off completely, that works as well and, so long as you have a home key on the keyboard, you can turn it back on quickly.

**Comment by bdorer** [comment]
yes, that's true, but the app or the readme should inform about this.
Maybe the App could provide a message telling the user to deactivate VO
or turning quicknavigation mode off.

**Comment by EscolarProgramming** [comment]
Thank your for this idea, it will be implemented in the next release!

**Comment by bdorer** [comment]
hmm, just an idea: Games are able to specify an area on the screen where VO passes all gestures through to the game. May you could use this for your App? If you don't know wich feature I mean I'll research.
